recipes/pocketsphinx/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pocketsphinx" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/cmusphinx/files/{{ name }}/5prealpha/{{ name }}-5prealpha.tar.gz
+  sha256: ef5bb5547e2712bdf571f256490ef42a47962033892efd9d7df8eed7fe573ed9
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - libtool
+    - swig
+    - sphinxbase
```

Comment:
I think sphinxbase should probably go in the host section, but that is something you can fix in the feedstock. 

Also is sphinxbase header-only or statically linked, because it is not present in the run dependencies?